tests/cxxopts.hpp

```diff
@@ -134,9 +134,9 @@ namespace cxxopts
 
```

Comment:
cxxopts.hpp is not "from me". Do we need to change it? I think no?

Reply:
You're right that changes to this file were excessive, at the very least. It seems to me that _some_ change was needed to resolve compiler errors with LLVM 17. I pinged you on a new comment about that.